src/main/java/mate/academy/spring/controller/UserController.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import mate.academy.spring.service.UserService;
+import mate.academy.spring.service.dto.mapping.DtoResponseMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+    private final UserService userService;
+    private final DtoResponseMapper<UserResponseDto, User> mapper;
+
+    public UserController(UserService userService,
```

Comment:
We need to use @Autowired  annotation above constructors everywhere or nowhere to make our code more consistent. Remove it everywhere or add it everywhere